Assignment1/src/asgn1/Assignment1.java

```diff
@@ -0,0 +1,79 @@
+//Assignment 1 completed
+package asgn1;
+
+import java.util.Scanner;
+
+public class Assignment1 {
+	public static void main(String args[]) {
+		var sc=new Scanner(System.in);
+		int i=1;
+		double tax=0;
+		double cost=0;
+//loop continues to take item input and give output until user chooses 'n'		
+		while (2>i) {
+		System.out.print("Enter an item: ");
+		String name=sc.next();
+		double price=0.0;
+		int quantity=0;
+		String type="raw";
+//Since the input order can be random. here we distinguish the type of the input and save it in a variable accordingly.
+//it is assumed that item price is entered as a double value and not an integer. 
+		for(int j=0;j<3;j++) {
+			if (sc.hasNextInt()) { quantity=sc.nextInt();}
+			else if (sc.hasNextDouble()) { price=sc.nextDouble();}
+			else if (sc.hasNext()) { type=sc.next();}
+		}
+		//System.out.println("The item is: " +name+"  price: "+price+"  type: "+type+"  quantity: "+quantity);
+
+//calculating tax for the item according to its type		
+		if (type.equalsIgnoreCase("raw")){
+			tax=calRawTax(price);
+			cost=(price+tax)*quantity;
+		}
+		else if (type.equalsIgnoreCase("manufactured")){
+			tax=calManufacturedTax(price);
+			cost=(price+tax)*quantity;
+		}
+		else if (type.equalsIgnoreCase("imported")){
+			tax=calImportedTax(price);
+			cost=(price+tax)*quantity;
+		}
+		else {
+			System.out.println("Invalid item type.");
+			continue;
+		}
```

Comment:
Consider leveraging java's inheritance property to make the code cleaner and scalable.